src/pages/index.js

```diff
@@ -44,12 +68,19 @@ function openProfilePopup() {
 
 function changeProfileInfo(event, data) {
   user.setUserInfo(data);
+  api.postUserInfo(data);
+  api.fetchUserInfo();
```

Comment:
`api.fetchUserInfo();` is not needed when you edit the profile data. It has no effect